resources/charts/kubernetes/templates/kubelet-ds.yaml

```diff
@@ -60,10 +60,6 @@ spec:
           mountPath: /dev
         - name: run
           mountPath: /run
-        # TODO check if this is needed
```

Comment:
`/sys` is needed for setting and reading cgroups.

Reply:
kept it.